docs/usage/advanced.rst

```diff
@@ -105,3 +105,14 @@ If you need to work with many Entity Sets the same way or if you just need to pi
 
     count = getattr(northwind.entity_sets, 'Employees').get_entities().count().execute()
     print(count)
+
+Avoid default values for nulled values
+--------------------------------------
+
+To avoid missing property values getting substituted by their (type specific) default values, the service can be
+configured to retain missing values.
+
+.. code-block:: python
+
+    count = getattr(northwind.entity_sets, 'Employees').get_entities().count().execute()
+    print(count)
```

Comment:
The documentation looks good except I am not sure how does the code sample relates to it?

Reply:
Not at all :face_with_spiral_eyes: - this slipped in by accident and the documentation part is still WIP. Would like to give a "real-world" example using Northwind, just need to find some entity which actually returns a null property.

Reply:
Fixed